doc-content/enterprise-only/openshift/con-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,8 @@
+[id="con-dashbuilder-standalone_{context}"]
+= Dashbuilder Standalone on {OPENSHIFT}
+
+Dashbuilder Standalone is an independent implementation of Dashbuilder, the dashboard and reporting tool integrated with {CENTRAL}.
```

Comment:
Suggest a minor rewrite to clarify: 

Dashbuilder Standalone is an independent implementation of Dashbuilder, which is the dashboard and reporting tool that is integrated with {CENTRAL}.

Reply:
Agreed, I would also phrase it as "that is".